R/uploadPackage.R

```diff
@@ -109,6 +124,26 @@ uploadPackageServer <- function(id) {
     )
     
     uploaded_pkgs00 <- reactiveVal()
+
+    observeEvent(user$role, {
+    req(user$role == "admin")  
+    output$rem_pkg_div <- renderUI({
+      column(
+        width = 4,
+      div(
+        id = "rem-package-group",
+        style = "display: flex;",
+        selectizeInput(NS(id, "rem_pkg_lst"), "Remove Package(s)", choices = NULL, multiple = TRUE,
+                       options = list(create = TRUE, showAddOptionOnCreate = FALSE, 
+                                      onFocus = I(paste0('function() {Shiny.setInputValue("', NS(id, "curr_pkgs"), '", "load", {priority: "event"})}')))),
+        # note the action button moved out of alignment with 'selectizeInput' under 'renderUI'
+        actionButton(NS(id, "rem_pkg_btn"), shiny::icon("angle-right"),
+                     style = 'height: calc(1.5em + 1.5rem + 2px); position: relative; top: 32px'),
```

Comment:
I was getting odd placement from this button. I was able to correct the issue by replacing `position: relative; top: 32px` with `margin-top: 32px`.

Also, can we change the button icon to maybe a trash can or an X?

Reply:
Fixed both of those.